### PR TITLE
Add Cloud page to Platform UI

### PR DESCRIPTION
**Pull request by goldenapples**
A working branch that enables the /cloud route in the platform-ui app, and adds the modules specified in https://github.com/humanmade/hm-cloud-plugin/issues.

Currently includes:
- [x] Pull Requests display https://github.com/humanmade/hm-cloud-plugin/issues/6
- [x] Environment Data display https://github.com/humanmade/hm-cloud-plugin/issues/19
- [ ] Contact Details Display (in https://github.com/humanmade/platform-ui/pull/16)
- [ ] Activity Log Display (in https://github.com/humanmade/platform-ui/pull/17)

### CODE REVIEW

**Comment by mikeselander** [comment]
@goldenapples it's _super_ out of date, but a god bit of this work has already been started in https://github.com/humanmade/platform-ui/pull/6. Might be useful for you to reference it if you're taking this over.

**Comment by goldenapples** [comment]
Oh nice, thanks @mikeselander. I like the enhancements to `withFetch` there to pass the credentials through, rather than having to do it in every component like I was doing. Might need to make a different HOC though now since withFetch is being used for a bunch of other APIs now. I'll crib a bunch of stuff from that branch since it's already got a lot of what I was trying to do done.

**Comment by goldenapples** [comment]
OK, I've pulled in some of the work from #6 and merged it in with the structure I had started with here. Could I get a review on this at your convenience? I'm unsure whether it makes more sense to merge this into master or keep it on a separate branch so that all of the cloud page UI can be merged at once. In the meantime, I'm opening new PRs for additional components on the cloud page against this branch (like #16).

Note: I added a DashboardWrapper component which I was planning on using to hold drag/drop and expand/close state for each of these widgets, until I saw the work in #14 which would be good to reuse for this page. So the DashboardWrapper component here is just minimal markup to inherit some core styles - I'm expecting it'll be replaced soon.

File: src/utils/withApiFetch.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+/* global HM */
+import withFetch from './withFetch';
+
+/**
+ * Fetch data from the local REST API, with credentials.
+ *
+ * Call withFetch(), with the API credentials passed in, and destructures the
+ * response for ease of using API data in the wrapped component.
+ */
+const withApiFetch = ( endpoint, options = {}, name = null ) => {
```
Comment (roborourke) [inline src/utils/withApiFetch.js]: I love composition :)

**Pull request merged**